### PR TITLE
Reconnect to RabbitMQ

### PR DESCRIPTION
**Pull request by rom10**

### CODE REVIEW

File: lib/connection.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ Connection.prototype.connect = function() {
 
     // Handle connection error
     conn.on("error", function(err) {
-      if (err.message !== "Connection closing") {
+      if (err.message == "read ECONNRESET") {
+        console.error("[AMQP] reconnecting");
+        setTimeout(that.connect.bind(that), 1000);
```
Comment (asafdav) [inline lib/connection.js]: move to parameter

File: lib/connection.js [hunk 2]
```diff
@@ -44,15 +48,18 @@ Connection.prototype.connect = function() {
...
         setTimeout(that.connect.bind(that), 1000);
       }
     });
 
-
   }, function(err) {
     console.error("[AMQP] conn error", err.message);
+    if (err.message.match(/ECONNREFUSED/)) {
+      console.error("[AMQP] reconnecting");
+      setTimeout(that.connect.bind(that), 5000);
```
Comment (asafdav) [inline lib/connection.js]: move to parameter

**Comment by asafdav** [comment]
@rom10 it looks great, let's demo and merge it back in

File: lib/connection.js (hunk 1 above)
Comment (rom10) [inline lib/connection.js]: :+1:

File: lib/connection.js (hunk 2 above)
Comment (rom10) [inline lib/connection.js]: :+1:

**Pull request merged**